kits/flutter_ume_kit_ui/lib/third_party/cyclop/lib/src/widgets/eyedrop/eye_dropper_layer.dart

```diff
@@ -10,7 +9,7 @@ import 'eye_dropper_overlay.dart';
 
 final captureKey = GlobalKey();
 
-class _EyeDropperModel {
+class UMEEyeDropperModel {
```

Comment:
The code which under the 'third_party/cyclop' is copied from third repository. Because the third-repo author didn't merge my changes. I think we shouldn't modify code in our repo. Should we release a new pub component to modify these code?